extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationOnSiteNotifications.tracking.js

```diff
@@ -0,0 +1,127 @@
+define('ext.wikia.design-system.on-site-notifications.tracking', [
+		'jquery',
+		'wikia.log',
+		'wikia.tracker',
+		'ext.wikia.design-system.on-site-notifications.common'
+	], function ($, log, tracker, common) {
+		'use strict';
+
+		function Tracking(model) {
+			this.model = model;
+			this.labels = {
```

Comment:
TBH that was a copy-paste from Mercury. Fixing, so that the dicts are used properly where it makes sense (markAsRead and markAllAsRead).